hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/FreqKList.java

```diff
@@ -0,0 +1,52 @@
+package org.hiero.sketch.spreadsheet;
+
+import org.hiero.sketch.dataset.api.IJson;
+import org.hiero.sketch.dataset.api.Pair;
+import org.hiero.sketch.table.RowSnapshot;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class FreqKList implements Serializable, IJson {
+    /**
+     * The size of the input table.
+     */
+    public final long totalRows;
+    /**
+     * The number of counters we store: the K in top-K heavy hitters.
+     */
+    public final int maxSize;
+    /**
+     * Estimate for the number of times each row in the above table occurs in the original DataSet.
+     */
+    public final HashMap<RowSnapshot, Integer> hMap;
+
+    public FreqKList(long totalRows, int maxSize, HashMap<RowSnapshot, Integer> hMap) {
+        this.totalRows = totalRows;
+        this.maxSize = maxSize;
+        this.hMap = hMap;
+    }
+
+    public int GetTotalCount() {
+        return this.hMap.values().stream().reduce(0, Integer::sum);
+    }
+
+    public int GetErrBound() {
```

Comment:
Good question. This parameter ought to be an integer. That comes from the proof of correctness though, it is not obvious.